pkg/log/kzerolog/kzerolog.go

```diff
@@ -108,13 +104,51 @@ func truncate(i interface{}, maxLen int) string {
 	return fmt.Sprintf(padFmtStr, s)
 }
 
+func (ctx *zeroLogContext) calculateSkipFrames() {
+	if ctx.skipFrames > 0 {
+		return
+	}
```

Comment:
```suggestion
func (ctx *zeroLogContext) calculateSkipFrames() int {
	if ctx.skipFrames > 0 {
		return ctx.skipFrames
	}
```

I feel like it would be cleaner for callers to call `calculateSkipFrames` and just use the return value.
WDYT?

Reply:
sure